scss/mixins/_baseline-default.scss

```diff
@@ -8,6 +8,10 @@ $baseline-default: (
   columns-small: 3,
   columns-medium: 6,
   columns-large: 12,
+  pushes: (12, 9, 6, 3),
+  pulls: (12, 9, 6, 3),
+  prefixes: (12, 9, 6, 3),
+  suffixes: (12, 9, 6, 3),
```

Comment:
I think it's better to NOT have these defaults. Have the implementator explicitly define his pushes, pulls, prefixes and suffixes.

I suggest to set all these to an empty list `()`

Reply:
Soooo.. Now that I'm re-reading the scope of the ticket again..

We don't need any prefixes/suffixes here now. Is it even possible/desirable to pre/suffix zones at all? And we're not using them now anywhere anyway